docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-2.md

```diff
@@ -325,7 +325,12 @@ Current workspace is "root:espw".
 ```
 ```shell
 kubestellar-where-resolver &
-sleep 45
+# wait until where-resolver is ready by continuing once SinglePlacementSlice resource is available
+sleep 10
+kubectl ws root:wmw-c
+while ! kubectl get SinglePlacementSlice &> /dev/null; do
```

Comment:
There should be two of them, if I recall correctly; might be worth showing a wait until there are two of them.

Reply:
OK, will check wmw-s too.